src/rendering/renderers/shared/view/View.ts

```diff
@@ -43,8 +42,6 @@ export interface View
     /** this is the AABB rectangle bounds of the view in local untransformed space. */
     bounds: BoundsData;
 
-    /** Adds the current bounds of this view to the supplied bounds */
-    addBounds: (bounds: Bounds) => void;
```

Comment:
`addBounds` is a public API so we shouldn't be removing it

Reply:
I think i can deprecate this one.